Assets/Scripts/Buildings/Upgrades/HealthUpgrade.cs

```diff
@@ -1,25 +1,21 @@
-using UnityEngine;
-
 namespace Buildings.Upgrades
 {
     public class HealthUpgrade : UpgradeBase
     {
-        [SerializeField] private int[] health;
-
         protected override void Awake()
         {
             base.Awake();
-            if (Building.MaxLevel < health.Length)
-                Building.MaxLevel = health.Length;
+            if (Building.MaxLevel < upgradeValues.Length)
```

Comment:
Wenn das jetzt für alle Upgrades gleich ist, kann es dann nicht auch in die base class?